Dockerfile-default-gpu

```diff
@@ -29,15 +29,20 @@ ARG APEX_GIT
 ARG APEX_PATCH
 RUN /tmp/det_dockerfile_scripts/install_apex.sh
 
+RUN /tmp/det_dockerfile_scripts/build_nccl.sh
+ARG HOROVOD_NCCL_LINK=STATIC
+ARG HOROVOD_NCCL_HOME=/tmp/det_nccl/build
```

Comment:
did we ever confirm that this actually makes horovod use the NCCL we've just built?

Reply:
Yes - it's the newer version, AND the CUDA version is an exact match (which is exactly why I wanted to avoid the packages from the repositories).